checker/tests/tainting/PolyReturn.java

```diff
@@ -0,0 +1,24 @@
+import org.checkerframework.checker.tainting.qual.PolyTainted;
+import org.checkerframework.checker.tainting.qual.Tainted;
+import org.checkerframework.checker.tainting.qual.Untainted;
+
+@SuppressWarnings({"inconsistent.constructor.type", "super.invocation.invalid"})
```

Comment:
Could this be moved to the constructor, to limit the scope?
(A comment why these are suppressed would also be useful.)